bonsai/ethereum/contracts/BonsaiTest.sol

```diff
@@ -54,7 +54,9 @@ abstract contract BonsaiTest is Test, BonsaiCheats {
             bonsaiTestRelay = new BonsaiTestRelay(vm.envOr("TEST_BONSAI_TEST_RELAY_EXPECTED_CHAIN_ID", uint256(31337)));
             bonsaiRelay = new BonsaiRelayQueueWrapper(bonsaiTestRelay);
         } else {
-            IRiscZeroVerifier verifier = new RiscZeroGroth16Verifier();
+            uint256 control_id_0 = vm.envUint("CONTROL_ID_0");
+            uint256 control_id_1 = vm.envUint("CONTROL_ID_1");
```

Comment:
This factoring makes it impossible to run `forge test` without setting those environment variables. It also does not provide any instructions for how the user should locate these parameters. Are you still working on that? I think we should make sure these are set automatically to a reasonable default before merging this PR

Reply:
Hey yes, im still working on that part. Wanted to see if anyone had a better way of doing thing